nautobot/core/settings.py

```diff
@@ -557,6 +512,20 @@
     "django_prometheus.middleware.PrometheusAfterMiddleware",
 ]
 
+
+LOGGING = {"loggers": {"nautobot": {}}}
+setup_structlog_logging(
+    LOGGING,
+    INSTALLED_APPS,
+    MIDDLEWARE,
+    log_level="DEBUG" if DEBUG else "INFO",
+    debug=DEBUG,
+    debug_db=is_truthy(os.getenv("NAUTOBOT_LOG_DEBUG_DB", "False")),
+    plain_format=is_truthy(os.getenv("NAUTOBOT_LOG_PLAIN", "False")),
```

Comment:
If these are only configurable via environment variables (not ideal!) then they should be documented in `nautobot/docs/user-guide/administration/configuration/optional-settings.md` under the "Environment-Variable-Only Settings" header.

Reply:
What is a better way to configure these? Note, we need to process these configuration settings into `LOGGING` Django settings attribute.

Reply:
Ideally they'd be settable directly via Python in any `nautobot_config.py` as well without needing to manipulate `os.environ`.